ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml

```diff
@@ -158,6 +162,45 @@ tests:
         requests:
           cpu: 100m
     workflow: optional-operators-ci-operator-sdk-aws
+- always_run: false
+  as: operator-node-density-heavy-25nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    dependencies:
+      OPERATOR_PR_IMAGE: pipeline:network-observability-operator
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "25"
+      COMPUTE_NODE_TYPE: m6i.4xlarge
+      ES_BENCHMARK_INDEX: prod-netobserv-datapoints*
+      ES_METADATA_INDEX: perf_scale_ci*
+      ES_TYPE: qe
+      EXTRA_FLAGS: --es-index=ripsaw-kube-burner --metrics-profile metrics-aggregated.yml
+      INSTALLATION_SOURCE: Source
```

Comment:
this var will need to be removed as, it's replaced in #70045 and default installation will be for "Source"